test/OpenTelemetry.Tests/Metrics/MetricApiTestsBase.cs

```diff
@@ -178,6 +178,36 @@ public void MetricDescriptionIsExportedCorrectly(string description)
         Assert.Equal(description ?? string.Empty, metric.Description);
     }
 
+    [Fact]
```

Comment:
Just to clarify and document the behavior, consider adding a test with duplicate meters.

```csharp
using var meter1 = new Meter("MyMeter", "1.0", new List<KeyValuePair<string, object>> { new("Key1", "Value1") });
using var meter2 = new Meter("MyMeter", "1.0", new List<KeyValuePair<string, object>> { new("Key2", "Value2") });
```

I think what will happen here is that measurements from an identical instrument from each of the meters will be exported with the tags from meter1, correct?

Reply:
added tests for this explicitly.